src/ir_def/ir.ml

```diff
@@ -232,13 +232,17 @@ let full_flavor () : flavor = {
   has_poly_eq = true;
 }
 
+type actor_type = {
+  transient_actor_type: Type.typ;
+  stable_actor_type: Type.typ
```

Comment:
I guess stable_actor_type is the record of initial stable field value (probably without mutability disinctions) - what  is transient_actor_type?

Reply:
Ah, ok, it's just the original actor type.